docs/conf.py

```diff
@@ -21,7 +21,7 @@
 copyright = '2020, MTG-UPF'
 author = 'MTG-UPF'
 
-
+master_doc = 'index'
```

Comment:
I'm confused about this, this shouldn't be needed. according to the documentation, this is already the default, so it should work without the change: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-master_doc
What version of sphinx are you using?

Reply:
1.3.6

Reply:
I updated to the version in requirements.txt and it seems to work now